src/EpicRoadmap/react/Components/EpicRoadmapView.tsx

```diff
@@ -54,6 +54,14 @@ class EpicRoadmapViewContent extends React.Component<IEpicRoadmapViewProps, IEpi
             return (
                 <Spinner size={SpinnerSize.large} className="loading-indicator" label="Loading..." />
             );
+        } else if (uiState === UIStatus.Error) {
```

Comment:
This case still is too generic. Like @navneet-g said we need to detect the levels of backlog and then show this message. It should be similar to the case UIStatus.NoTeamIterations below.